### PR TITLE
Add a coffeelint.json

### PR DESCRIPTION
**Pull request by Arcanemagus**
Add a `coffeelint.json` and fix a few minor code style issues exposed by it.

[Review on Reviewable](https://reviewable.io/reviews/atomlinter/linter-pylint/72)

### CODE REVIEW

**Comment by SpainTrain** [comment]
Where did this config file come from?  Would be awesome if it were the same at atom-linter or something. Otherwise LGTM 


---

Reviewed 2 of 2 files at r1.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/atomlinter/linter-pylint/72)

**Comment by Arcanemagus** [comment]
I copied it from `linter-php` (I think, one of the ones I've put that on at least). I think the original source was a `coffeelint --makeconfig` that I then modified to match... something (Possibly the current state of `linter`?).

If you want to just use [this one](https://github.com/atom-community/linter/blob/master/coffeelint.json) that sounds perfectly fine to me.

**Comment by SpainTrain** [comment]
Slight preference to using https://github.com/atom-community/linter/blob/master/coffeelint.json, but frankly it's 6 vs. 0.5 doz.

File: coffeelint.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  "newlines_after_classes": {
+    "value": 3,
+    "level": "ignore"
+  },
+  "no_backticks": {
+    "level": "error"
+  },
+  "no_debugger": {
+    "level": "error",
+    "console": false
```
Comment (Arcanemagus) [inline coffeelint.json]: Not present in linter's.

File: coffeelint.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    "value": 2,
+    "level": "error"
+  },
+  "line_endings": {
+    "level": "ignore",
+    "value": "unix"
+  },
+  "max_line_length": {
+    "value": 120,
+    "level": "warn",
```
Comment (Arcanemagus) [inline coffeelint.json]: Set to `ignore` in linter's.

**Comment by Arcanemagus** [comment]
Commented on the differences, if you still want to use `linter`s I'll switch it over to that, but personally I would go with this one.

**Comment by SpainTrain** [comment]
:+1:

**Pull request merged**